docs/installation.md

```diff
@@ -21,6 +21,11 @@ Kowl Business supports Authentication and Authorization. In order to configure O
 
 While Kowl was written to be deployed in Kubernetes it does not have any specific dependencies. You can run it with any other container orchestrator or on bare metal.
 
+### Hosting and Proxies
+
+If you want to, you can host Kowl under a sub path (so for example `domain.com/some/sub/path` instead of `domain.com/`).
+See the [this document](./config/hosting.md) for more information.
```

Comment:
```suggestion
If you want to host Kowl under a sub path (e.g. `domain.com/some/sub/path` instead of `domain.com/`) take a look at [/docs/config/hosting.md](./config/hosting.md) for more information.
```